src/components/tab/SearchInput.vue

```diff
@@ -0,0 +1,308 @@
+<template>
+	<div id="search-bar">
+		<div class="input-field">
+			<input
+				id="workpackages-search"
+				v-model="search"
+				:placeholder="placeholder"
+				type="text"
+				@keyup="makeSearchRequest">
+		</div>
+		<div v-if="searchResults.length>0" class="search-list">
+			<div v-for="workPackage in searchResults" :key="workPackage.id" class="workPackage-item">
+				<div class="wp-info">
+					<div class="wp-info__filter-wp">
+						<div class="filter-project-type-status">
+							<div class="filter-project-type-status__project">
+								{{ workPackage.project }}
+							</div>
+							<div class="filter-project-type-status__type" :style="{'color':workPackage.typeCol}">
+								{{ workPackage.typeTitle }}
+							</div>
+							<div class="filter-project-type-status__status"
+								:style="{'background-color':workPackage.statusCol}">
+								{{ workPackage.statusTitle }}
+							</div>
+						</div>
+						<div v-if="workPackage.assignee" class="filter-assignee">
+							<div class="filter-assignee__avatar">
+								<Avatar
+									class="item-avatar"
+									:size="25"
+									:url="workPackage.picture"
+									:user="workPackage.assignee"
+									:display-name="workPackage.assignee" />
+							</div>
+							<div class="filter-assignee__assignee">
+								{{ workPackage.assignee }}
+							</div>
+						</div>
+					</div>
+					<div class="filter-wp-subject">
+						<div class="filter-wp-subject__subject">
+							{{ workPackage.subject }}
+						</div>
+					</div>
+				</div>
+			</div>
+			<div class="create-new-wp">
+				{{ t('integration_openproject', '+ New work package in OpenProject') }}
+			</div>
+		</div>
+		<div v-if="state === 'no-token' || state === 'error' || state === 'loading' || state === 'empty'"
```

Comment:
would that also work?
```suggestion
		<div v-if="state !== 'ok'"
```